### ISSUE TITLE
Application is crashing if a chat with a custom theme is opened

### ISSUE THREAD

**Issue by ndanilin** [telegramdesktop/tdesktop#27759]
### Steps to reproduce

1. Launch Telegram Desktop
2. Open a chat with a custom theme (this can be seen from the logs).

### Expected behaviour

The Chat is opened and all functions of an application are working.

### Actual behaviour

Telegram Desktop is closed unexpectedly.

### Operating system

Linux Mint 21.2 Cinnamon

### Version of Telegram Desktop

4.16.7

### Installation source

Flatpak

### Logs

```shell
[2024.04.16 09:49:43] Assertion Failed! "image.format() == QImage::Format_ARGB32_Premultiplied || image.format() == QImage::Format_RGB32" chat_theme.cpp:775
```

**Comment by GooseBumpsOS** [comment]
Same problem

**Comment by GooseBumpsOS** [comment]
duplicate issue https://github.com/telegramdesktop/tdesktop/issues/27745

**Comment by ndanilin** [comment]
duplicate issue https://github.com/telegramdesktop/tdesktop/issues/27745

**Comment by john-preston** [comment]
This should be fixed in 4.16.8 released now.

**Comment by hashiyomu** [comment]
it's a known QT frame issue
